src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,170 @@
+
+**Note to validator operators**: If you did not provide a withdrawal address on initial deposit, you will need to upgrade withdrawal credentials before any withdrawals will be enabled for your validator account. The [Staking Launchpad](https://launchpad.ethereum.org/withdrawals) has more details on when and how to do this.
+
+## Staking rewards {#staking-rewards}
+
+**Rewards payments** are automatically processed for active validator accounts with a maxed out effective balance of 32 ETH, who are accumulating rewards.
+
+Any balance above 32 ETH earned through rewards does not actually contribute to principle, or increase the sway of this validator on the network, and is thus automatically withdrawn as a rewards payment every few days. Aside from providing a withdrawal address one time, these rewards do not require any action from the validator operator, nor do they consume gas (no associated transaction fee) or take up existing block space.
+
+### How did we get here? {#how-did-we-get-here}
+
+Over the past few years Ethereum has undergone several network upgrades transitioning to a network secured by ETH itself, instead of energy-intensive mining as it once was. Participating in consensus on Ethereum is now known as "staking", as participants have voluntarily locked up ETH, placing it "at stake" for the ability to participate in the network. Following the rules rewards users, while attempts to cheat can be penalized.
+
+Since the launch of the staking deposit contract in November 2020, some brave Ethereum pioneers have voluntarily locked funds up to enable "validators", or accounts that have the right to formally attest to and propose blocks, following network rules.
+
+Before the enabling of staking withdrawals, none of the funds staked had any ability to be unlocked and used elsewhere. With the addition of staking withdrawals, staked ETH no longer requires an unknown lock-up period. Liquid rewards will automatically be deposited into the provided account, and stakers can unlock and withdrawn their entire balance at any time.
+
+### How do I prepare? {#how-do-i-prepare}
+
+<WithdrawalsTabComparison />
+
+### Important notices {#important-notices}
+
+Providing a withdrawal address is a required step for any validator account before it will be eligible to have ETH withdrawn from its balance.
+
+<InfoBanner emoji="⚠️" isWarning>
+  <strong>Each validator account can only be assigned a single withdrawal address, one time.</strong> Once an address is chosen and submitted to the Beacon Chain, this cannot be undone or changed again. Double-check ownership and accuracy of the address provided before submitting.
+</InfoBanner>
+
+Failure to update withdrawal credentials will simply leave the ETH locked in the validator account as it has been until a withdrawal address is provided. There is <strong>no threat to your funds in the meantime</strong> for not providing this—only the loss of opportunity.
```

Comment:
```suggestion
Failure to add withdrawal credentials will simply leave the ETH locked in the validator account as it has been until a withdrawal address is provided. There is <strong>no threat to your funds in the meantime</strong> for not providing this—only the loss of opportunity.
```
Add vs update, given this action can only be done once, correct?

Reply:
Correct, only once. Either way works for me. Per chat with @konopkja, we're going to switch the order of these two sentences as well

Reply:
```suggestion
There is <strong>no threat to your funds in the meantime</strong> for not providing this—only the loss of opportunity. Failure to add withdrawal credentials will simply leave the ETH locked in the validator account as it has been until a withdrawal address is provided.
```